content/intro-to-storybook/react/en/data.md

```diff
@@ -105,24 +106,39 @@ PureTaskList.defaultProps = {
   loading: false,
 };
 
-export default connect(
-  ({ tasks }) => ({
-    tasks: tasks.filter(t => t.state === 'TASK_INBOX' || t.state === 'TASK_PINNED'),
-  }),
-  dispatch => ({
-    onArchiveTask: id => dispatch(archiveTask(id)),
-    onPinTask: id => dispatch(pinTask(id)),
-  })
-)(PureTaskList);
+export function TaskList() {
+  // We're retrieving our state from the store
+  const tasks = useSelector(state => state.tasks);
```

Comment:
:hand: Imports on line 87 need to be changed from `import { connect }` to `import { useSelector, useDispatch }`

Reply:
Updated with the latest commit. Thanks for the speedy review and help us catch the error we really appreciate it!